pxl_scripts/README.md

```diff
@@ -36,12 +36,11 @@ Visualize these in three separate time series charts.
 - px/[pod_lifetime_resource](https://github.com/pixie-labs/pixie/tree/main/pxl_scripts/px/pod_lifetime_resource): Total resource usage of a pod over it's lifetime.
 - px/[pod_memory_usage](https://github.com/pixie-labs/pixie/tree/main/pxl_scripts/px/pod_memory_usage): Get the Virtual memory usage and average memory for all processes in the k8s cluster.
 - px/[pods](https://github.com/pixie-labs/pixie/tree/main/pxl_scripts/px/pods): List of Pods monitored by Pixie in a given Namespace with their high level application metrics (latency, error-rate & rps) and resource usage (cpu, writes, reads).
-- px/[postgres_data](https://github.com/pixie-labs/pixie/tree/main/pxl_scripts/px/postgres_data): Shows most recent PGSQL (Postgres) messages in the cluster.
-- px/[psql_data](https://github.com/pixie-labs/pixie/tree/main/pxl_scripts/px/psql_data): Shows a sample of PostgreSQL request in the cluster.
+- px/[psql_data](https://github.com/pixie-labs/pixie/tree/main/pxl_scripts/px/psql_data): Shows most recent PGSQL (Postgres) messages in the cluster.
```

Comment:
Can we name this `pgsql` instead of `psql`?

Reply:
This readme file is autogenerated from the script manifests. Just want to confirm that you want me to rename the `psql*` pxl scripts since this could potentially break someone's API setup.

Reply:
@oazizi000 - I will do the renaming in a separate PR when I add `px/pgsql_flow_graph` and update `px/pgsql_stats` to account for `trace_role`.